spec/services/broadcasts/welcome_notification/generator_spec.rb

```diff
@@ -10,10 +18,34 @@
     end
 
     context "when sending a welcome_thread notification" do
-      xit "generates the appropriate broadcast to be sent to a user"
-      xit "it sends a welcome notification for that broadcast"
-      xit "it does not send duplicate welcome notification for that broadcast"
-      xit "does not send a notification to a user who has commented in a welcome thread"
+      let(:welcome_thread_article) { create(:article, title: "Welcome Thread - v3000", published: true) }
```

Comment:
Do we actually need this `v3000` here? The `3000` seems a bit random to me and could be confusing down the road. Why not just `v0`?

Reply:
We do not need it to be any particular number--I have made this change!